components/ws-manager/pkg/manager/headless.go

```diff
@@ -115,16 +120,24 @@ func (hl *HeadlessListener) handleLogLine(pod *corev1.Pod, line string) (continu
 	return true
 }
 
-type theiaLogMessage struct {
-	Message   theiaTaskLogMessage `json:"message"`
-	Component string              `json:"component"`
-}
-
-type theiaTaskLogMessage struct {
+type taskLogMessage struct {
 	Type string `json:"type"`
 	Data string `json:"data"`
 }
 
+type workspaceLogMessage struct {
+	taskLogMessage
+	Component string `json:"component"`
+}
+
+//region backward compatibility
```

Comment:
@csweichel do we need backward compatibility here? Can it happen that old Theia is running with new ws-manager?

Reply:
That depends a bit when we deploy. If we bring this into prod with the next deployment, then no this cannot happen.

If we deploy this thereafter, then yes, it can.

Reply:
ok, will keep it just to be safe